config/config.go

```diff
@@ -368,6 +419,34 @@ type Log struct {
 	RecordPlanInSlowLog uint32     `toml:"record-plan-in-slow-log" json:"record-plan-in-slow-log"`
 }
 
+// Instance is the section of instance scope system variables.
+type Instance struct {
+	// These variables only exist in [instance] section.
+
+	// GeneralLog is used to log every query in the server in info level.
+	GeneralLog bool `toml:"tidb_general_log" json:"tidb_general_log"`
```

Comment:
How about renaming `GeneralLog` to `TiDBGeneralLog` to match the the serialized field name?

/cc @morgo

Reply:
Sure, this makes sense to me too.